docs/sql-ref-ansi-compliance.md

```diff
@@ -72,16 +72,23 @@ The type conversion of Spark ANSI mode follows the syntax rules of section 6.13
 
 | Source\Target | Numeric | String | Date | Timestamp | Interval | Boolean | Binary | Array | Map | Struct |
 |-----------|---------|--------|------|-----------|----------|---------|--------|-------|-----|--------|
-| Numeric   | Y       | Y      | N    | N         | N        | Y       | N      | N     | N   | N      |
-| String    | Y       | Y      | Y    | Y         | Y        | Y       | Y      | N     | N   | N      |
+| Numeric   | <span style="color:red">Y</span> | Y      | N    | N         | N        | Y       | N      | N     | N   | N      |
```

Comment:
For following the universal design, could we use a bold char or `*` instead? Or use the combination of them?